Frameworks/include/CoreGraphics/DWriteWrapper.h

```diff
@@ -35,7 +35,7 @@ static inline CFStringRef _CFStringCreateUppercaseCopy(CFStringRef string) {
     return ret;
 }
 
-COREGRAPHICS_EXPORT CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localizedString);
+COREGRAPHICS_EXPORT CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localizedString, CFStringRef* language);
```

Comment:
CFStringRef* language = nullptr
This saves you from all the extra null addition changes.